aea/cli/config.py

```diff
@@ -54,9 +55,13 @@ def config(click_context: click.Context) -> None:  # pylint: disable=unused-argu
 @pass_ctx
 def get(ctx: Context, apply_environment_variables: bool, json_path: str) -> None:
     """Get a field."""
+    if apply_environment_variables:
+        aev_flag_depreaction()
+
     try:
         agent_config_manager = AgentConfigManager.load(
-            ctx.cwd, apply_environment_variables
+            ctx.cwd,
+            substitude_env_vars=True,
```

Comment:
We set this true by default from now on, we remove this argument as it might lead to some breaking changes. We will safely remove this pre V2 release